src/clj/rems/workflow/permissions.clj

```diff
@@ -0,0 +1,145 @@
+(ns rems.workflow.permissions
+  (:require [clojure.set :as set]
+            [clojure.test :refer [deftest is testing]]))
+
+(def ^:private conj-set (fnil conj (hash-set)))
```

Comment:
What is the purpose of this def?

Reply:
To produce a set when conj'ing nil.

Reply:
I would rather put it to utils or sth with more descriptive name.

Reply:
I think the name is already as descriptive as it is, and it's such a rarely needed operation that moving it elsewhere only complicates reading the code.

Reply:
I think this is fine as it is now.